### DISCUSSION TITLE
Inconsistency in KDM-BA between code and formula

### DISCUSSION THREAD

**Discussion by yellowbridge** [dayoonkwon/BioAge#17] [Q&A] [answered]
Thanks for your insight papers,   
"Accelerated biological aging and risk of depression and anxiety: evidence from 424,299 UK Biobank participants" and 
"A toolkit for quantification of biological age from blood chemistry and organ function test data: BioAge".
Both papers provide the formula to evaluate the biological aging process and its impact on health.

When we try to understand its basic theory and application, we get confused about the calculation of KDM-BA prediction.
In the formula, 
![image](https://github.com/dayoonkwon/BioAge/assets/14272202/db75aa2b-61d6-4136-b72d-6ed5d71b7101)

In the BioAge package and corresponding `kdm_calc`  function,
![image](https://github.com/dayoonkwon/BioAge/assets/14272202/97741ad7-8861-4c39-819d-1819ddc51c5d)

There is an inconsistency between the formula and the R code, we are wondering which one is correct.

**Comment by dayoonkwon** [comment, marked as answer]
It's an error in the paper. "k" is the slope and "q" is the intercept as stated in the BioAge packages's `kdm_calc` function and [Klemera and Doubal 2006](https://www.sciencedirect.com/science/article/pii/S0047637405002654?via%3Dihub).
